### DISCUSSION TITLE
Display the script file name at the top of the script screen

### DISCUSSION THREAD

**Discussion by TweedleDim** [godotengine/godot-proposals#9509] [Editor]
Version 4.2.1

The title speaks for itself. I guess it would be quite simple to add and would make it clearer which script is displayed in the editor. You no longer need to search the side panel to find which script is highlighted, as the file name can be found in the top center.

![screenshot](https://github.com/godotengine/godot-proposals/assets/104669791/9b97785e-ca32-4e43-9d6a-07e151f5efe1)

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/5461.
